iml-agent/systemd-units/rust-iml-agent.service

```diff
@@ -13,6 +13,8 @@ Environment=CRT_PATH=/etc/iml/self.crt
 Environment=PFX_PATH=/etc/iml/identity.pfx
 Environment=AUTHORITY_CRT_PATH=/etc/iml/authority.crt
 Environment=LTUER_CONF_PATH=/etc/iml/ltuer.conf
+# Here {name} corresponds to the field names of iml_agent::action_plugins::lamigo::Config
+Environment=LAMIGO_UNIT_PATH=/etc/systemd/system/lamigo-{fs}-{mdt}.service
```

Comment:
From the Michael's proposal to make the path to be the part of the environment.

I've decided not to do just `LAMIGO_DIR=/etc/systemd/system`, since it is one of the standard systemd locations and to have it configurable looks redundant.
Additionally to this, we might want to configure the name of the unit itself. Therefore the full path is configurable and the comment above.